packages/math-input/src/components/keypad/keypad-pages/extras-page.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+
+import Keys from "../../../data/key-configs";
+import Key from "../../../data/keys";
+import {KeypadButton} from "../keypad-button";
+
+type Props = {
+    extraKeys: ReadonlyArray<Key>;
+    onClickKey: (keyConfig: string) => void;
```

Comment:
Should this be typed as `ClickKeyCallback`?

Reply:
Good catch